src/components/RichRadio/RichRadio.tsx

```diff
@@ -78,22 +78,36 @@ export type RichRadioProps = {
   onChange: (newVal: string) => void;
   items: RichRadioItemType[];
   value: string;
+  selectedIcon?: 'check' | 'right-chevron';
```

Comment:
What do you think about making this more flexible by passing in the desired icon?
```suggestion
  icon?: ReactNode;
```